explorer/types.go

```diff
@@ -596,3 +596,35 @@ type HostQuery struct {
 	AcceptContracts      *bool             `json:"acceptContracts,omitempty"`
 	Online               *bool             `json:"online,omitempty"`
 }
+
+// MarshalJSON implements json.Marshaler.  The embedded types.SiacoinInput
+// in our SiacoinInput has its own marshaler that will override default
+// marshaling and result in fields we expect being missing.
+func (e SiacoinInput) MarshalJSON() ([]byte, error) {
```

Comment:
Probably prefer to remove the embed over this

Reply:
Yeah it's not great, but adding all the fields of the currently embedded type to the explorer types might be preferable here.